src/Sylius/Behat/Context/Api/Admin/ManagingShippingMethodsContext.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Client\ResponseCheckerInterface;
+use Webmozart\Assert\Assert;
+
+final class ManagingShippingMethodsContext implements Context
+{
+    /** @var ApiClientInterface */
+    private $client;
+
+    /** @var ResponseCheckerInterface */
+    private $responseChecker;
+
+    public function __construct(ApiClientInterface $client, ResponseCheckerInterface $responseChecker)
+    {
+        $this->client = $client;
+        $this->responseChecker = $responseChecker;
+    }
+
+    /**
+     * @When I want to browse shipping methods
+     */
+    public function iBrowseShippingMethods(): void
+    {
+        $this->client->index();
+    }
+
+    /**
+     * @Then I should see :count shipping methods in the list
+     */
+    public function iShouldSeeShippingMethodsInTheList(int $count): void
+    {
+        Assert::count($this->responseChecker->getCollection($this->client->index()), $count);
```

Comment:
Maybe it should be `getLastResponse()`? I think we should use it in the steps when the previous `index` call is natural ("in the list", "in the registry" etc.), wdyt?